mne/epochs.py

```diff
@@ -147,8 +147,6 @@ def _save_part(fid, epochs, fmt, n_parts, next_fname, next_idx):
     # save baseline
     if epochs.baseline is not None:
         bmin, bmax = epochs.baseline
-        bmin = epochs.times[0] if bmin is None else bmin
-        bmax = epochs.times[-1] if bmax is None else bmax
```

Comment:
Not needed anymore because we replace `None` with `timin` and `tmax` early on now